### DISCUSSION TITLE
Is a DAW plugin version planned?

### DISCUSSION THREAD

**Discussion by svavilov** [GrandOrgue/grandorgue#1047] [General]
is it possible to create an LV2/VST/AU plugin with the same functionality as the standalone software?

**Comment by oleg68** [comment]
No, there are no such plans.

If you need the capability to integrate GrandOrgue with other sound processing applications, I'd suggest you to use Jack Audio Connection Kit.

**Comment by BuzzAgain** [comment]
I think a DAW plugin solution would be a massive workflow improvement that would explode the GrandOrgue user base exponentially.

**Reply by larspalo** [reply]
I personally don't think it would be any improvement for the core GrandOrgue at all. GrandOrgue is a virtual instrument (a very specific sample player that tries to emulate real pipe organs) that can very well be used to record tracks in Ardour or whatever. JACK is the way to go.

**Reply by SimonDundas** [reply]
Excuse me but I'm not very technical!   I'm also not a very good keyboard player but i like to compose.   I use the Cubase DAW to record midi,  maker any corrections, then send the corrected midi to the virtual instrument.    The Hauptwerk vitual organ software can do this but the VST link is now only available on the top,  expensive version.   GrandOrgue has many wonderful organ sample sets I'd like to work with.   Will the Jack Audio Connection Kit enable me to do what I want?   thanks in advance for your expertise.

**Reply by MStraeten** [reply]
exploding user base isn't a value itself for a free open source software ;)
If thats a such important feature then there'd be developers stepping in and start coding. But usually there are plenty of good ideas but no one contributing to implement these.
So with a limited deveopler base the scope needs to kept on what can be delivered in quality ....

**Reply by larspalo** [reply]
@SimonDundas Unfortunately I've no experience with Cubase (either on Windows or Mac) myself as I almost exclusively use various Linux distributions with Ardour (or sometimes Rosegarden in the past) instead for any kind of DAW work where JACK indeed enables any kind of routing imaginable between different softwares.

Unless someone with actual experience of using Cubase under Windows/Mac can chime in and help, the best option for you is just to try it out yourself and experiment. Feel free to start a specific thread on the subject to share your experiences doing it which could very well inspire others in turn!

From the plugin point of view, I've heard from other users that GrandOrgue is more or less considered to be overkill when used as a VST as it's perhaps too detailed and resource intensive (depending on what sample set is used). But even if GrandOrgue is mostly intended to be a stand alone instrument don't let that discourage you from test using GO with Cubase!

**Reply by SimonDundas** [reply]
Thanks Larspalo.    Since Jack handles midi as well as audio data that could probably link GrandOrgue and Cubase.   If it requires any programming I'll probably be out of my depth!

**Comment by oleg68** [comment]
I agree with @larspalo. Jack allows to integrate GrandOrgue with other musical software.

**Comment by chmaha** [comment]


**Reply by larspalo** [reply]
@chmaha Feel absolutely free to use whatever you think is appropriate for the situation! If you're not happy using GrandOrgue as a pipe organ simulator, do use something else instead that you feel does the trick better.

**Comment by midimusic** [comment]
I agree with lars and oleg. I do not use DAW's and the ones I have tried have never done  anything useful to me. It seems to me that they are mainly intended for making the horrible noises in the background of most pop music. I do not use Jack either all it usually does is cause trouble.
csw900

**Comment by RustoMCSpit** [comment]
if this were planned, a .clap would be nice

**Comment by BuzzAgain** [comment]
This rabbit hole for an organ virtual instruments is absurd and bloated beyond belief. I recommend the elegant, efficient and small footprint "Aeolus" plugin found here for free ... https://github.com/Archie3d/aeolus_plugin

**Reply by larspalo** [reply]
You're absolutely free to use whatever floats your boat! :rofl:
